### DISCUSSION TITLE
The game should load a saved face in a new game automatically if there is one.

### DISCUSSION THREAD

**Discussion by Leo512bit** [OpenTTD/OpenTTD#10047] [Features]
Surprised it's been missing for so long.
(Sorry if I came across as rude didn't mean do, just genuinely surprised.)

**Comment by glx22** [comment]
Already implemented with #9895.

**Reply by 2TallTyler** [reply]
Implemented but not yet released. 😉

To use this feature, play a Nightly version (available in Steam using Properties->betas or the OpenTTD website) or wait for the next release.

**Reply by Leo512bit** [reply]
>Implemented but not yet released

I checked to see if anyone had some other discussion about it but failed to check the nightly builds. I'm dumb.😅

Also another thing that's mildly annoying when creating new games is there's no option to set a default company and CEO name. It would be nice to have something in the ottd config. This time I checked the nightly build and I didn't see it :P

**Reply by 2TallTyler** [reply]
Yeah that doesn't exist yet, but as usual...Pull Requests accepted. 😉

**Reply by Leo512bit** [reply]
> Yeah that doesn't exist yet, but as usual...Pull Requests accepted. 😉

If I can get over my fear of large codebases then maybe😄
